### PR TITLE
Clean up the Mango RC upgrade handlers.

### PR DESCRIPTION
**Pull request by dwedul-figure**
## Description

This PR is currently just to get a run of the release workflow so we have something to test our credentials with.

For some reason, github didn't fire off the checks when it was in Draft, but it's not a PR that's ready to be merged.

---

Before we can merge this PR, please make sure that all the following items have been
checked off. If any of the checklist items are not applicable, please leave them but
write a little note why.

- [x] Targeted PR against correct branch (see [CONTRIBUTING.md](https://github.com/provenance-io/provenance/blob/main/CONTRIBUTING.md#pr-targeting))
- [ ] Linked to Github issue with discussion and accepted design OR link to spec that describes this work.
- [ ] Wrote unit and integration [tests](https://github.com/provenance-io/provenance/blob/main/CONTRIBUTING.md#testing)
- [ ] Updated relevant documentation (`docs/`) or specification (`x/<module>/spec/`)
- [ ] Added relevant `godoc` [comments](https://blog.golang.org/godoc-documenting-go-code).
- [ ] Added a relevant changelog entry to the `Unreleased` section in `CHANGELOG.md`
- [x] Re-reviewed `Files changed` in the Github PR explorer
- [ ] Review `Codecov Report` in the comment section below once CI passes

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -37,28 +37,11 @@ Ref: https://keepachangelog.com/en/1.0.0/
 
 ## Unreleased
 
-## [v1.11.1-rc4](https://github.com/provenance-io/provenance/releases/tag/v1.11.1-rc4) - 2022-06-23
+## [v1.11.1](https://github.com/provenance-io/provenance/releases/tag/v1.11.1) - 2022-06-28
```
Comment (dwedul-figure) [inline CHANGELOG.md:40]: This needs to be updated to the current date before this PR is merged.

**Review by arnabmitra — DISMISSED** [review]
lgtm ..both SetParams and RunMigrations are set in mango handler

**Review by arnabmitra — DISMISSED** [review]

**Review by iramiller — DISMISSED** [review]

File: CHANGELOG.md [hunk 2]
```diff
@@ -37,28 +37,13 @@ Ref: https://keepachangelog.com/en/1.0.0/
...
-
-### Bug Fixes
-
-* Add new `msgfees` `NhashPerUsdMil`  default param to param space store on upgrade (PR [#875](https://github.com/provenance-io/provenance/issues/875))
-* Add `mango-rc2` upgrade handler to do a couple migration pieces that were missed in v1.11.1-rc1, but needed [PR 888](https://github.com/provenance-io/provenance/pull/888).
-
----
-
-## [v1.11.1-rc1](https://github.com/provenance-io/provenance/releases/tag/v1.11.1-rc1) - 2022-06-14
+## [v1.11.1](https://github.com/provenance-io/provenance/releases/tag/v1.11.1) - 2022-07-12
```
[the remark this answers is not in the document]
Comment (iramiller) [reply CHANGELOG.md]: Looks like it has been updated to the 13th

**Review by derekadams — DISMISSED** [review]

File: CHANGELOG.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (iramiller) [reply CHANGELOG.md]: I don't think we should have dates in here like this ... the git history provides these dates.  Further the date may not be accurate in all global contexts anyways.

[the remark this answers is not in the document]
Comment (dwedul-figure) [reply CHANGELOG.md]: I'm torn on this. It's nice to see it on the releases page, and it's really only a pain to get right anymore due to not being able to merge into the `.x` branch without a PR. Usually we're not adding this line until we're ready to actually cut the release. I just lumped it in with this PR to make fewer PRs required.

**Review by arnabmitra — APPROVED** [review]

**Pull request merged**